README.md

```diff
@@ -78,3 +79,32 @@ aws ssm get-parameters-by-path --path /aws/service/aws-for-fluent-bit/
 ```
 
 For more see [our docs](https://github.com/aws/aws-for-fluent-bit#public-images).
+
+
+### KPL aggregation
+
+KPL aggregation can be enabled by setting the `aggregation` parameter to `true` (default is false).  With aggregation enabled records will be serialized into the KCL protobuf structure containing a batch of records before being sent via PutRecords.  This batch of records will only count as a single record towards the Kinesis records per second limit (currently 1000 records/sec per shard).
```

Comment:
nit: With aggregation enabled each Record in the PutRecords request can contain multiple serialized records in the KCL protobuf structure.

Reply:
(My suggested rewording of your sentence)